openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -257,6 +257,17 @@ def verify_identity_url(self):
             else:
                 return IDVerificationService.get_verify_location('verify_student_verify_now', self.course_key)
 
+    @property
+    def verification_status(self):
+        """
+        Returns a String of the verification status of learner.
+        """
+        if self.enrollment_object and self.enrollment_object.mode in CourseMode.VERIFIED_MODES:
+            return IDVerificationService.user_status(self.effective_user)['status']
```

Comment:
user_status is guaranteed to return an object with a `status` key?

Reply:
Yessir